cmd/waku/node.go

```diff
@@ -382,15 +382,13 @@ func Execute(options NodeOptions) error {
 	var wg sync.WaitGroup
 
 	if options.Relay.Enable {
-		for nodeTopic := range pubSubTopicMap {
+		for nodeTopic, cTopics := range pubSubTopicMap {
 			nodeTopic := nodeTopic
-			sub, err := wakuNode.Relay().SubscribeToTopic(ctx, nodeTopic)
+			_, err := wakuNode.Relay().Subscribe(ctx, wprotocol.NewContentFilter(nodeTopic, cTopics...))
 			if err != nil {
 				return err
 			}
 
-			sub.Unsubscribe()
-
 			if len(options.Rendezvous.Nodes) != 0 {
 				// Register the node in rendezvous point
```

Comment:
We're creating a subscription here and not reading it. Wouldn't that mean that once the msg channel buffer is filled, we'd end up with a "subscriber too slow" error? I wonder if we should have an option in `Subscribe` to indicate that we're not interested in the messages themselves and that translates into a 
```
for _ := range mySub.Ch {
   // Do nothing
}
```

Reply:
Ah, you are right.
That is a good idea, will add that.